packages/common/src/store/pages/saved-page/types.ts

```diff
@@ -21,9 +21,24 @@ export function isLibraryCategory(value: string): value is LibraryCategoryType {
   return Object.values(LibraryCategory).includes(value as LibraryCategoryType)
 }
 export interface SavedPageState {
-  localSaves: { [id: number]: UID }
+  localTrackFavorites: { [id: number]: UID }
+  localTrackReposts: { [id: number]: UID }
+  localTrackPurchases: { [id: number]: UID }
```

Comment:
It's a tad confusing that tracks use an object mapping from IDs-> UIDs but I suppose this pattern was already here and you're getting it functional again?

Reply:
Yeah exactly, prob worth another revisit in the future

Reply:
I'm not entirely sure that the UID thing is necessary (prob is but wasn't obvious)